command.go

```diff
@@ -101,6 +101,13 @@ type CommandFunc struct {
 	// See Command for details about the accepted signatures.
 	Func interface{}
 
+	// An optional usage string for this function. If set, then this replaces the
+	// the default one that shows the types (but not names) of arguments.
+	Usage string
+
+	// Set of options to not set from the environment
+	IgnoreEnvOptions map[string]struct{}
```

Comment:
How about making this a list of strings? It would allow for a more user-friendly syntax. We can convert to a map internally to make efficient lookups.
```suggestion
	IgnoreEnvOptions []string
```